test/OpenTelemetry.AutoInstrumentation.Tests/Configurations/SettingsTests.cs

```diff
@@ -142,19 +142,39 @@ internal void SdkSettings_DefaultValues()
     [InlineData("otlp,none", new[] { TracesExporter.Otlp })]
     [InlineData("non-supported,none", new TracesExporter[0])]
     [InlineData("zipkin,non-supported,none", new[] { TracesExporter.Zipkin })]
-    internal void TracesExporter_SupportedValues(string tracesExporter, TracesExporter[] expectedTracesExporter)
+    internal void TracesExporter_SupportedValues(string tracesExporters, TracesExporter[] expectedTracesExporters)
     {
-        Environment.SetEnvironmentVariable(ConfigurationKeys.Traces.Exporter, tracesExporter);
+        Environment.SetEnvironmentVariable(ConfigurationKeys.Traces.Exporter, tracesExporters);
 
         var settings = Settings.FromDefaultSources<TracerSettings>(false);
 
-        settings.TracesExporters.Should().Equal(expectedTracesExporter);
+        settings.TracesExporters.Should().Equal(expectedTracesExporters);
     }
 
-    [Fact]
-    internal void TracesExporter_FailFast()
+    [Theory]
+    [InlineData("non-supported", new TracesExporter[0])]
+    [InlineData("otlp,otlp", new[] { TracesExporter.Otlp })]
+    [InlineData("otlp, ,console", new[] { TracesExporter.Otlp, TracesExporter.Console })]
+    [InlineData("otlp, , console", new[] { TracesExporter.Otlp })]
+    [InlineData("otlp,,,", new[] { TracesExporter.Otlp })]
+    internal void TracesExporter_FailFastOff(string tracesExporters, TracesExporter[] expectedTracesExporters)
```

Comment:
Why not adding test cases to `TracesExporter_SupportedValues`? It already has a similar case `[InlineData("non-supported,none", new TracesExporter[0])]`

Reply:
Because it’s not supported values, this cases have different errors or warnings, but it can work when failfast is disabled

Reply:
The test logic is exact the same. We can also rename the `TracesExporter_SupportedValues` to something else. Maybe even `internal void TracesExporter` (without the suffix).

Reply:
Okey, i make it with renaming